src/styles/main.scss

```diff
@@ -0,0 +1,5 @@
+@import './vars';
```

Comment:
Ensure that the `vars` file contains definitions for all variables used across the imported SCSS files, such as `$card-blue-color`, `$nav-link-color`, and `$blue-color`, to avoid any compilation issues.